### DISCUSSION TITLE
force delete order?

### DISCUSSION THREAD

**Discussion by bmaster001** [pretix/pretix#3931] [Q&A] [answered]
I was playing with a new event, and forgot to put it in test-mode before I placed an order... now I am unable to delete the order, nor the event. Is there any trick to delete orders that were entered in non-test mode?

**Comment by pc-coholic** [comment, marked as answer]
Without going into the database and manipulating the data in there (at your own peril): No.

pretix is designed on purpose to be auditable - making it possible to delete supposedly undeletable data is exactly the opposite of that promise ;-)
